src/main/java/com/uc/BoredNoMore/dao/IActivityDAO.java

```diff
@@ -18,6 +18,12 @@
 	 */
 	void addActivity(ActivityDTO activity);
 	
+	/**
+	 * Save activity to database
+	 * @return void
+	 */
+	void saveActivity(ActivityDTO activity);
```

Comment:
For some reason when I created this code review branch and cloned it to my PC, there was an addActivity method but not saveActivity so I updated it. I would still update this file regardless of which method was missing